packages/story-editor/src/components/panels/design/pageBackground/pageBackground.js

```diff
@@ -114,6 +128,15 @@ function PageBackgroundPanel({ selectedElements, pushUpdate }) {
     })
   );
 
+  useEffect(() => {
+    getFailingPages(pages, pageSize, currentPage).then((failedPages) => {
+      // getFailingPages returns an array of pages, since we only care
+      // about currentPage, we can grab the single page result.
+      const result = failedPages[0]?.result;
+      setFailedContrast(result?.length ? true : false);
+    });
+  }, [currentPage, pageSize, pages]);
```

Comment:
🤔 I think we don't need this effect to run when `pages` changes. However, if we use the `getFailingPages` function like this:

```getFailingPages([currentPage], pageSize)```

then this effect won't be run when `pages` changes